homeassistant/components/zwave_js/light.py

```diff
@@ -184,7 +187,7 @@ def color_mode(self) -> str | None:
     @property
     def is_on(self) -> bool:
         """Return true if device is on (brightness above 0)."""
```

Comment:
I don't know how nit-picky things are around here, but if you're making another change, may as well update the comment since it's outdated now.